testsuite/MDAnalysisTests/formats/test_libdcd.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import print_function
+
+from nose.tools import raises
+from numpy.testing import assert_equal, assert_array_equal
+from numpy.testing import assert_array_almost_equal
+from numpy.testing import assert_almost_equal
+
+from MDAnalysis.lib.formats.libdcd import DCDFile
+from MDAnalysisTests.datafiles import DCD
+
+from unittest import TestCase
+from MDAnalysisTests.tempdir import run_in_tempdir
+import numpy as np
+
+class DCDReadFrameTest(TestCase):
+
+    def setUp(self):
+        self.dcdfile = DCDFile(DCD)
+
+    def tearDown(self):
+        del self.dcdfile
+
+    def test_read_coords(self):
+        # confirm shape of coordinate data against result from previous
+        # MDAnalysis implementation of DCD file handling
+        dcd_frame = self.dcdfile.read()
+        xyz = dcd_frame[0]
```

Comment:
The `dcd_frame` is a namedtuple. You can access the coordinates also with `dcd_frame.x`